### PR TITLE
Diagram database

### PR DESCRIPTION
**Pull request by sumairq**
## In this pull request we added the following:
 
- Add new tables.
- Convert diagrams and arrows connections into tables.
- Use Many to Many relationships.

## File added 
`schema_based_on_diagram.sql`

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	type VARCHAR(255),
+	name VARCHAR(255),
+	PRIMARY KEY(id)
+);
+
```
Comment (BenMukebo) [inline schema_based_on_diagram.sql:22]: - [ ] Great job so far 👏🏻 👏🏻 , However  All "join table" for the many-to-many relationships should be created, even if they are not showed on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different)

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- All tables have a PRIMARY KEY set :heavy_check_mark:
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] This is a group project. Please let's include all the authors in the README file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+CREATE TABLE invoice_items(
+	id INT GENERATED ALWAYS AS IDENTITY,
+	unit_price DECIMAL,
+	quantity INT,
+	total_price DECIMAL,
+	invoices_id INT,
+	treatment_id int,
+	PRIMARY KEY(id)
+);
```
Comment (BenMukebo) [inline schema_based_on_diagram.sql:31]: - [ ] Great job so far 👏🏻 👏🏻 , However  All "join table" for the many-to-many relationships should be created, even if they are not showed on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different)

Comment (sumairq) [reply schema_based_on_diagram.sql:31]: Thank you! for feedback.

**Review by codepantha — APPROVED** [review]
Hi team 👋, 
## STATUS: APPROVED ✔️ 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**